mlos_bench/mlos_bench/tests/config/schemas/environments/test-cases/good/full/local_fileshare_env-full.jsonc

```diff
@@ -29,6 +29,8 @@
         "dump_params_file": "/tmp/dump_params.json",
         "dump_meta_file": "/tmp/dump_params_meta.json",
 
+        "parse_results_stdout": "(\\w+),([0-9.]+)",
+
         "read_results_file": "/tmp/results.csv",
```

Comment:
Are these mutually exclusive?

Also, does parse results stdout belong in fileshare services, or just the scriptenv (remote/local exec)?

Reply:
No, currently we just merge the results from stdout and the file when both parameters are present.

Parsing the results does not belong to the fileshare services - e.g. RemoteEnv has no access to fileshare (only the LocalFileShareEnv does), but we might want to parse its stdout